bentoml/adapters/file_input.py

```diff
@@ -101,6 +101,20 @@ def predict(self, file_stream):
             <input type="submit">
         </form>
 
+    OR by python requests:
+
+    .. code-block:: python
+
+        import requests
+
+        with open("test.jpg", "rb") as f:
+            image_bytes = f.read()
```

Comment:
Similarly let's give the user another example to turn a PIL image object into a request?